docs/platform/howto/use-azure-privatelink.rst

```diff
@@ -161,22 +161,69 @@ To enable Private Link access in `Aiven Console <https://console.aiven.io/>`_:
 
     Each service component can be controlled separately. For example, you can enable Private Link access for your Aiven for Apache Kafka® service, while allowing Kafka® Connect to only be connected via VNet peering.
 
-After toggling the values your Private Link resource will be rebuilt with load balancer rules added for the service component's ports. Connection information like the URI or hostname and port to access the service through the private endpoint is available on the service's **Overview** page in `Aiven Console <https://console.aiven.io/>`_. 
+After toggling the values, your Private Link resource will be rebuilt with load balancer rules added for the service component's ports.
 
 .. note::
   
   For Aiven for Apache Kafka® services, the security group for the VPC endpoint must allow ingress in the port range ``10000-31000``. This is to accommodate the pool of Kafka broker ports used in the Private Link implementation.
 
+Acquire connection information
+------------------------------
+
+.. _one-privatelink-connection:
+
+One Azure Private Link connection
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you have Azure Private Link enabled for one service component, you can preview its connection information (URI, hostname, or port required to access the service through the private endpoint) on the service's **Overview** page in `Aiven Console <https://console.aiven.io/>`_.
+
+Multiple Azure Private Link connections
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you have more than one Azure Private Link connection, you can get connection information for the first endpoint as described in :ref:`One Azure Private Link connection <one-private-service-connection>` (from `Aiven Console <https://console.aiven.io>`__). For connection information for the remaining endpoints, you need to use CLI.
```

Comment:
should we use cloud provider terminology or ours? for instance "Azure private endpoint" rather than "Azure Private Link connection". 

Also ```:ref:`One Azure Private Link connection <one-private-service-connection>` ``` links to Google PSC.

Reply:
We can use both. I believe the users are aware with the concept of Azure Private Link at this point in this article so introducing term "Azure private endpoint" and using both in parallel should not confuse them.